config/core/deployments/pingsource-jobrunner.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2018 The Knative Authors
```

Comment:
New deployment for the pingsource jobrunner. This is to be included in the manifest.